### PR TITLE
CTED-2519

### PR DESCRIPTION
**Pull request by BrendonPretorius**
Rework simple nav to only load selected cats at the start

### CODE REVIEW

**Comment by BrendonPretorius** [comment]
Initial code has large amount of errors so only added relevant lines for screenshots
Screenshot 2022-08-23 at 07 04 58
Screenshot 2022-08-23 at 07 04 41

File: block_simple_nav.php [hunk 1]
```diff
@@ -242,6 +243,12 @@ public function get_content() {
             return $this->content;
         }
 
+        if (empty($this->config)) {
+            $this->content->text = get_string('configurecats', 'block_simple_nav');
+            return $this->content;
+        }
+
+
```
Comment (Mmahadinana) [inline block_simple_nav.php]: There is two new lines here.

**Review by Mmahadinana — APPROVED** [review]

File: block_simple_nav.php [hunk 2]
```diff
@@ -250,9 +256,8 @@ public function get_content() {
 
         // get all the categories and courses from the navigation node
         // save them in $this->categories
```
Comment (JJSwanevelder) [inline block_simple_nav.php:258]: Is this comment still valid given the code below?

**Review by JJSwanevelder — APPROVED** [review]
👍

**Pull request merged**